charts/zookeeper/values.yaml

```diff
@@ -26,6 +26,12 @@ config: {}
   # syncLimit: 2
   # quorumListenOnAllIPs: false
 
+ephemeral:
+  enabled: false
+  #emptydirvolumesource:
```

Comment:
don't comment `emptydirvolumesource`
Instead set it to
```
   emptydirvolumesource: {}
      #medium: ""/Memory
      #sizeLimit: 15Gi
```
If user wishes to provide a value he can remove `{}` and uncomment the options

Reply:
Done